src/main/java/opendota/Parse.java

```diff
@@ -307,6 +314,8 @@ else if (cle.getType() == DOTA_COMBATLOG_TYPES.DOTA_COMBATLOG_GOLD) {
             else if (cle.getType() == DOTA_COMBATLOG_TYPES.DOTA_COMBATLOG_XP) {
                 combatLogEntry.xp_reason = cle.getXpReason();
             }
+            combatLogVisitors.visit(combatLogEntry);
```

Comment:
You determine two properties based on the combat log entry and current parse state, right?

Can you just return an object that contains `tracked_death` and `greevils_greed_stack`?